docs/zope4/migration/zodb.rst

```diff
@@ -154,12 +152,12 @@ Going from Python 2 to Python 3
   ``bin/zodbupdate -n -f var/filestorage/Data.fs --convert-py3 --encoding utf-8 --encoding-fallback latin1``
```

Comment:
This step will currently fail. When using dry run, there is no automatic ZODB magic number rewrite, so when the database file is opened the ZODB code will error out with magic numbers older than `FS30`. Dry run on an older ZODB only works on Python 2 right now.